lib/conduit_lwt_server.mli

```diff
@@ -1,6 +1,8 @@
 
 val close : 'a Lwt_io.channel * 'b Lwt_io.channel -> unit Lwt.t
 
+val set_max_active : int -> unit
```

Comment:
Is there any way to do this on a per server basis? This doesn't seem like a constant that can be universal for all servers started with conduit.